Directory.Build.targets

```diff
@@ -1,25 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-    <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
 
-    <ItemGroup>
-        <PackageReference Update="System.Diagnostics.Process" Version="4.3.0" />
-        <PackageReference Update="System.IO.Compression" Version="4.3.0" />
-        <PackageReference Update="System.Memory" Version="4.4.0-preview1-25305-02" />
-        <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-        <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
+  <ItemGroup>
+    <PackageReference Update="System.Diagnostics.Process" Version="4.3.0" />
+    <PackageReference Update="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Update="System.Memory" Version="4.4.0-preview1-25305-02" />
+    <PackageReference Update="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
 
-        <PackageReference Update="Microsoft.DotNet.Cli.CommandLine" Version="$(MicrosoftDotNetCliCommandLinePackageVersion)" />
-        <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-        <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
-        <PackageReference Update="xunit" Version="2.3.1" />
-        <PackageReference Update="xunit.runner.visualstudio" Version="2.3.1" />
-    </ItemGroup>
+    <PackageReference Update="Microsoft.DotNet.Cli.CommandLine" Version="$(MicrosoftDotNetCliCommandLinePackageVersion)" />
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
+    <PackageReference Update="xunit.abstractions" Version="2.0.3" />
+  </ItemGroup>
 
-    <ItemGroup Condition="'$(TestProject)' == 'true'">
-        <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Do we know what this was doing? Is it safe to remove?

Reply:
Looks like something VS2013 did... Not needed anymore: https://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for

Reply:
This is also included to all tests project by SDK Arcade - we don' need it here.